src/pages/index.astro

```diff
@@ -136,7 +136,7 @@ const latest = (await releases)[0];
         href="https://geopandas.org/en/stable/docs/reference/api/geopandas.read_parquet.html">writing</a> GeoParquet.
       </li>
       <li>
-        <a href="https://qgis.org">QGIS</a> Windows and Linux ship with GeoParquet support, and Mac can work installing with <a href="https://docs.conda.io/en/latest/">conda</a> (from the terminal with conda activated run 'conda install qgis libgdal-arrow-parquet', and then just type 'qgis' in the terminal). 
+        <a href="https://qgis.org">QGIS</a> Windows and Linux ship with GeoParquet support, and Mac can work installing with <a href="https://docs.conda.io/en/latest/">conda</a> (from the terminal with conda activated run 'conda config --add channels conda-forge', 'conda install qgis libgdal-arrow-parquet', and then just type 'qgis' in the terminal).
```

Comment:
Instead of globally adding the `conda-forge` channel, it should also work to use `conda install -c conda-forge` as well, though I'll let others debate the merits of each approach